ibis/backends/pandas/kernels.py

```diff
@@ -349,6 +347,24 @@ def arbitrary(arg):
     ops.Log: lambda x, base: np.log(x) if base is None else np.log(x) / np.log(base),
 }
 
+
+def none_proof(func):
```

Comment:
```suggestion
def none_safe(func):
```

I think safe is a much more common word to use for this functionality.